tests/unit/conftest.py

```diff
@@ -267,23 +261,19 @@ def flat_mix_status_xml(tmpdir_factory):
             name="test_fail[ansible://localhost]" time="0.00335693359375">
                 {testcase_properties}
                 <failure message="assert False">host = &lt;testinfra.host.Host object at 0x7f0921d98cd0&gt;
-
             def test_fail(host):
         &gt;       assert False
         E       assert False
-
         tests/test_default.py:18: AssertionError</failure>
             </testcase>
             <testcase classname="tests.test_default" file="tests/test_default.py" line="20"
             name="test_error[ansible://localhost]" time="0.00208067893982">
                 {testcase_properties}
                 <error message="test setup failure">host = &lt;testinfra.host.Host object at 0x7f0921d98cd0&gt;
-
             @pytest.fixture
             def error_fixture(host):
         &gt;       raise RuntimeError(&apos;oops&apos;)
         E       RuntimeError: oops
-
         tests/test_default.py:10: RuntimeError</error>
             </testcase>
             <testcase classname="tests.test_default" file="tests/test_default.py" line="24"
```

Comment:
You stripped a lot of newline spacing in this file, are you sure you want to do that?